### ISSUE TITLE
AnnotationUseStyle does not detect all violations in EXPANDED style

### ISSUE THREAD

**Issue by strkkk** [checkstyle/checkstyle#6818]
```
$ cat TestClass.java
public class TestClass {
    @MyAnnotation(@Deprecated) // violation
        class Example1 {}
        @MyAnnotation({}) // violation
        class Example2 {}
        @MyAnnotation(@Deprecated()) // violation
        class Example3 {}

        @interface MyAnnotation {
        Deprecated[] value();
        }
}

$ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="FinalLocalVariable" />

<module name="AnnotationUseStyleCheck">
<property name="elementStyle" value="expanded" />
<property name="closingParens" value="ignore" />
</module>
    </module>
</module>

$ java -jar checkstyle-8.20-all.jar -c config.xml TestClass.java
Starting audit...
Audit done.
```

Expected: violations on lines commented with `// violation`

**Comment by rnveach** [comment]
Fix was merged
